src/App.vue

```diff
@@ -17,23 +19,51 @@ import editor from "./components/editor";
 import viewer from "./components/viewer";
 
 export default {
-  name: 'App',
+  name: "App",
+  props: {
+    appData: {
+      type: Object,
+      required: true,
+    },
+  },
   components: {
     notes,
     editor,
     viewer,
   },
   data() {
     return {
-      notesDate: ["1","2",3],
-
+      currentNote: {
+        title: "000",
+        favorite: false,
+        content: "ZZZZZZZ",
+        id: "0",
+      },
+      notes: [
+        {
+          title: "111",
+          favorite: false,
+          content: "oneoneone",
+          id: "1",
+        },
+        {
+          title: "222",
+          favorite: false,
+          content: "twotwotwo",
+          id: "2",
+        },
+      ],
+    };
+  },
+  methods: {
+    tap(id) {
+      this.currentNote = this.notes[id];
     }
   },
-}
+};
 </script>
 
 <style lang="scss">
```

Comment:
I will add `scoped` to make sure that this style won't affect other components.
```suggestion
<style lang="scss" scoped>
```